docs/connecting-to-a-lit-network/migrating-to-datil.md

```diff
@@ -58,9 +58,10 @@ The only code changes required to make use of the new Datil networks are as foll
 
 - Upgrade the Lit packages to the latest version that supports Datil
   - `6.2.2` is the minimum version of the packages that support `datil-test` and `datil-dev`
```

Comment:
```suggestion
  - `6.4.0` is the minimum version of the packages that support `datil-test` and `datil-dev`
```

Reply:
I do think `6.2.2` was the first to support these networks, at latest `6.3.0`. Do we want to update this to `6.4.0`?

Reply:
Changed this to `6.3.0` for the testnets